client/src/components/Forms/BodyCheck.js

```diff
@@ -125,6 +125,21 @@ class BodyCheck extends Component {
     this.setState(stateObj);
   };
 
+  handleFieldInputDate = (event) => {
```

Comment:
Export this function from a utility file if one doesn’t exist already. Then use it on all of the other existing files. Of course you will have to pass and return the updated state.

that way it’s only defined in one place